internal/entimport/import.go

```diff
@@ -62,6 +62,8 @@ func WithTables(tables []string) ImportOption {
 type SchemaImporter interface {
 	// SchemaMutations imports a given schema from a data source and returns a list of schemast mutators.
 	SchemaMutations(context.Context) ([]schemast.Mutator, error)
+	// field receives an Atlas column and converts in to an ent field.
+	field(column *schema.Column) (f ent.Field, err error)
```

Comment:
This interface signature looks a bit odd. Why is one method exported and the other private ?

Reply:
I don't need the "field" function exported, the other option is to create a type for "field" and pass or make it exported.
wdyt?

Reply:
@rotemtam see last commit, extracted the func.